src/Umbraco.Core/Persistence/PetaPocoSqlExtensions.cs

```diff
@@ -34,11 +34,27 @@ public static Sql Where<T>(this Sql sql, Expression<Func<T, bool>> predicate)
             return sql.Where(whereExpression, expresionist.GetSqlParameters());
         }
 
+        private static string GetFieldName<T>(Expression<Func<T, object>> fieldSelector, ISqlSyntaxProvider sqlSyntax)

```

Comment:
No big deal it's fine as it is. Just caught me out when doing a quick glance over at first.